pom.xml

```diff
@@ -71,7 +64,7 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-inline</artifactId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
Mockito recently stopped shipping `mockito-inline` as a separate artifact. It is now the default implementation shipped in `mockito-core`.

Reply:
Too bad, I liked to have it as a separate artifact since it was usually a code smell to have to use it :)
Anyway thanks!